lib/tasks/report_primary_organisation.rake

```diff
@@ -0,0 +1,79 @@
+namespace :report_primary_organisation  do
+  desc "A one-off CSV report of how many documents were initially published by authors who don't belong to the document's primary organisation"
+  task :generate_csv, [:user_email] => :environment do |t, args|
```

Comment:
Could be worth checking for the presence of the `user_email` argument and failing fast if it's not there (or even doesn't look like an email address?)